package.json

```
@@ -40,19 +40,23 @@
 		"geth:start": "WEB3_SYSTEM_TEST_BACKEND=geth && ./scripts/geth.sh start",
 		"geth:start:background": "WEB3_SYSTEM_TEST_BACKEND=geth && ./scripts/geth.sh start 1",
 		"geth:stop": "./scripts/geth.sh stop",
+		"ipc:start:background": "WEB3_SYSTEM_TEST_BACKEND=ipc && ./scripts/ipc.sh start 1",
+		"ipc:stop": "./scripts/ipc.sh stop",
 		"lint": "lerna run lint --stream --parallel",
 		"lint:fix": "lerna run lint:fix --stream --parallel",
 		"format": "lerna run format --stream --parallel",
 		"test": "lerna run test --stream --parallel",
 		"test:unit": "lerna run test:unit --stream --parallel",
 		"test:integration": "lerna run test:integration --stream",
+		"test:integration:ipc": "cd ./packages/web3 && yarn test:integration test/integration/ipc.test.ts",
 		"test:e2e:ganache:http": "./scripts/test-runner.sh ganache http",
 		"test:e2e:ganache:ws": "./scripts/test-runner.sh ganache ws",
 		"test:e2e:geth:http": "./scripts/test-runner.sh geth http",
 		"test:e2e:geth:ws": "./scripts/test-runner.sh geth ws",
 		"test:e2e:ganache:ws:electron": "./scripts/test-runner.sh ganache ws electron",
 		"test:e2e:ganache:ws:chrome": "./scripts/test-runner.sh ganache ws chrome",
 		"test:e2e:ganache:ws:firefox": "./scripts/test-runner.sh ganache ws firefox",
+		"test:e2e:ipc": "./scripts/test-runner.sh ipc ipc ipc",
```

Would be nice to mention the backend, mode and runner. 

```suggestion
		"test:e2e:geth:ipc": "./scripts/test-runner.sh geth ipc electron",
```

---

for IPC it works only with the geth binary backend (not docker). I called it ipc. Regarding runner. may be yes we can run it from browsers and  electron